### PR TITLE
libprelude: Update to 5.2.0

### PR DESCRIPTION
**Pull request by ToToL**
#### Description

Bump version 5.2.0

###### Type(s)
- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Verification
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: security/libprelude/Portfile [hunk 1]
```diff
@@ -3,10 +3,10 @@
 PortSystem      1.0
 
 name            libprelude
-version         5.1.1
+version         5.2.0
 revision        1
```
Comment (ryandesign) [inline security/libprelude/Portfile:7]:
```suggestion
revision        0
```

**Review by ryandesign — CHANGES_REQUESTED** [review]
Fails:
```
patching file configure.ac
Hunk #1 FAILED at 489.
Hunk #2 succeeded at 523 (offset 20 lines).
Hunk #3 succeeded at 781 (offset 20 lines).
1 out of 3 hunks FAILED -- saving rejects to file configure.ac.rej
```

**Comment by ryandesign** [comment]
Still fails:
```
patching file configure.ac
Reversed (or previously applied) patch detected!  Assume -R? [n] 
Apply anyway? [n] 
Skipping patch.
1 out of 1 hunk ignored -- saving rejects to file configure.ac.rej
```

**Comment by ToToL** [comment]
Hello

It is now OK

I also integrated your review about "revision" field

Regards

File: security/libprelude/Portfile (hunk 1 above)
Comment (ToToL) [reply security/libprelude/Portfile]: Done

File: security/libprelude/Portfile [hunk 2]
```diff
@@ -3,10 +3,10 @@
 PortSystem      1.0
 
 name            libprelude
-version         5.1.1
-revision        1
+version         5.2.0
+revision        0
 categories      security
-license         GPL-2+
+license         LGPL-2+
```
Comment (ryandesign) [inline security/libprelude/Portfile:9]:
```suggestion
license         LGPL-2.1+
```

File: security/libprelude/Portfile [hunk 3]
```diff
@@ -67,12 +64,6 @@ destroot.keepdirs \
         ${destroot}${prefix}/var/spool/prelude \
         ${destroot}${prefix}/etc/prelude/profile
 
-variant python27 description {python2.7 bindings} {
-        configure.args-replace  --without-python2 \
-                                --with-python2=${prefix}/bin/python2.7
-        depends_lib-append port:python27
-}
```
Comment (ryandesign) [inline security/libprelude/Portfile:74]: Is it no longer possible to build python27 bindings?

Comment (ToToL) [reply security/libprelude/Portfile:74]: Pyython 2 is EOL so I remove the support.

File: security/libprelude/Portfile (hunk 2 above)
Comment (ToToL) [reply security/libprelude/Portfile]: Done

**Pull request merged**